README.md

```diff
@@ -27,11 +27,11 @@ or manually:
 ```sh
 git clone https://github.com/yonaskolb/Mint.git
 cd Mint
-swift run mint install yonaskolb/mint
+mint install yonaskolb/mint
```

Comment:
Hey, thank you for the contribution!
Are you sure this part is correct? Cause at this step we only install Mint and `mint` command is not global yet.